ui/desktop/src/config.ts

```diff
@@ -53,7 +53,7 @@ export const extendGoosed = async (config: SystemConfig) => {
   }
 
   try {
-    const response = await fetch(getApiUrl('/systems/add'), {
+    const response = await fetch(getApiUrl('/extensions/add'), {
```

Comment:
this will make a small conflict - I think can leave this for now?

Reply:
ill remove and we can get @alexhancock 's PR in instead

Reply:
kewl